snipsskills/templates/README.rst

```diff
@@ -0,0 +1,39 @@
+{{project_name}} skill for Snips
+======================================
+
+|MIT License|
+
+{{project_name}}
+
+Installation
+------------
+
+Usage
+-----
+Snips Skills Manager
+^^^^^^^^^^^^^^^^^^^^
+
+It is recommended that you use this skill with the `Snips Skills Manager <https://github.com/snipsco/snipsskills>`_. Simply add the following section to your `Snipsfile <https://github.com/snipsco/snipsskills/wiki/The-Snipsfile>`_:
+
+Standalone usage
+^^^^^^^^^^^^^^^^
+
+Contributing
+------------
+
+Please see the `Contribution Guidelines`_.
+
+Copyright
+---------
+
+See `LICENSE.txt`_ for more information.
```

Comment:
This should also be removed (we can't force any license)

Reply:
hmmm, ok i'll remove it !